modules/virt-defining-a-watchdog.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-a-watchdog.adoc
+
+[id="virt-defining-a-watchdog"]
+= Defining a watchdog device
+
+.Procedure
+
+. Choose the watchdog action. This defines how the watchdog proceeds when the operating system (OS) no longer responds.
+
++
+.Available actions
+[horizontal]
+`poweroff`:: The virtual machine (VM) powers down, and the guest OS cannot react.
+`reset`:: The VM reboots in place, and the guest OS cannot react. Because the length of time required for the guest OS to reboot can cause liveness probes to timeout, you should avoid their use.
+`shutdown`:: The VM powers off gracefully and becomes non-runnable.
+
+. Create a YAML file and apply it to the cluster.
++
+The example below configures the i6300esb watchdog device on a Fedora VM with the poweroff action and exposes the device as `/dev/watchdog`.
++
+[source,yaml]
+----
+---
+apiVersion: kubevirt.io/v1
+kind: VirtualMachineInstance
```

Comment:
I think that giving an example with a VirtualMachine (not a throw-away VMI) would be more helpful for users